### PR TITLE
Auto-redirect is not working

### PR DESCRIPTION
**Pull request by guanbinrui**
Fix the auto-redirect not working.

More: [Workflow 1B: Core Init / Restore Backup](https://confluence.dimension.chat/pages/viewpage.action?pageId=2064444)

### CODE REVIEW

File: src/extension/options-page/Home.tsx [hunk 1]
```diff
@@ -98,8 +98,6 @@ export default function DashboardHomePage() {
     )
 
     React.useEffect(() => {
-        if (personas.length !== 0) return
```
Comment (SunriseFox) [inline src/extension/options-page/Home.tsx:101]: I have no idea about why...

Comment (guanbinrui) [reply src/extension/options-page/Home.tsx:101]: An unconnected persona should goto **1A[6] "Connect Profile"** when restoring the backup.

Comment (SunriseFox) [reply src/extension/options-page/Home.tsx:101]: In fact 0 means `the data is not ready` so the following check logic will always execute at least once. Removing this line won't ensure the check logic more (but make the query happens every time the router change).

Comment (guanbinrui) [reply src/extension/options-page/Home.tsx:101]: Actually, we need to check if the persona is connected when every time the router changes.

**Pull request merged**